# cluster
seg_distance: [20, 40, 60]
cluster_distance: [0.6, 1.0, 1.3, 1.8]
leaf_size: 0.15
min_cluster_size: 10
max_cluster_size: 2000
